bids/README.md

```diff
@@ -23,41 +23,62 @@ The neurospin_to_bids.py script can be use without install the unicog module.
 The test_dataset follows a minimal set of conventions for the download to work.
 
 
-# Using:
+# Basic usage:
 
-        cd path_to/test_dataset
+        cd <path_to/test_dataset>
         python path_to/neurospin_to_bids.py -neurospin_database trio
 
+In this case, the **\<path_to/test_dataset\>** directory contains a **exp_info** directory with 2 files 
+(**participants.tsv** and **download.tsv**) indicating what are the subjects to download.
+For more information, read the Additional information section.
+Indicate your **neurospin_database** either "prisma" or "trio" (prisma is the default parameter).
+
+# Advanced usage:
+
+        python neurospin_to_bids.py -root_path <root_path> -dataset_name <my_dataset> -neurospin_database trio
+
+The **-root_path** option allows to indicate a specific path including the **exp_info** directory.
+The **-dataset_name** option allows to give a specific name to the dataset ( the default is bids_dataset).
+For more information, read the Additional information section.
 
 # Additionnal information
 ## Input files for the preparation of importation:
-Two input files are needed:
-- participants.tsv: corresponds to the list of participants. Here is an example:
+Two input files are mandatory and the events are optionals:
+
+- **participants.tsv**: corresponds to the list of participants. Here is an example:
 
         participant_id	NIP	acq_date
         sub-1	nip_number	2010-06-28
         sub-2	nip_number	2010-07-01
```

Comment:
it should be sub-02 instead of sub-2 in the example. Its important to clarify that everything after the dash is interpreted as a string. Also to recommend to use padded numbers (02 instead of 2) for automatic ordering.